walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -86,14 +79,15 @@
       TimeRange currentEvent = clashAttendeeEvents.get(i);
       TimeRange followingEvent = clashAttendeeEvents.get(i+1);
 
-      if (!(currentEvent.contains(followingEvent)) && !(currentEvent.overlaps(followingEvent))) {
-        // case 1
+
+      if (!(currentEvent.overlaps(followingEvent))) {
+         // case 1
         TimeRange gap = TimeRange.fromStartEnd(currentEvent.end(), followingEvent.start(), false);
         long possibleDuration = gap.duration();
         if (meetingDuration <= possibleDuration) {
           proposedTimeRanges.add(gap);
         }
-      } else if (!currentEvent.contains(followingEvent) && currentEvent.overlaps(followingEvent)) {
+      } else if (currentEvent.overlaps(followingEvent)) {
         // case 2
         // do nothing - not an eligible time
       } else if (currentEvent.contains(followingEvent) && !currentEvent.overlaps(followingEvent)) {
```

Comment:
nit: please also remove the check for `contains` from here too for consistency :)